pyproject.toml

```diff
@@ -30,6 +30,7 @@ rich = "^12.6.0"
 dataclasses-json = "^0.5.7"
 validators = "^0.20.0"
 oauth2-client = "^1.3.0"
+cinder = "^21.1.0"
```

Comment:
mkdocs-cinder is already in the dev dependencies. It should not be needed by users of the SDK.